ktor-io/jvm/src/io/ktor/utils/io/ByteBufferChannel.kt

```diff
@@ -27,7 +29,7 @@ internal open class ByteBufferChannel(
 ) : ByteChannel, ByteReadChannel, ByteWriteChannel, LookAheadSuspendSession, HasReadSession, HasWriteSession {
 
     // internal constructor for reading of byte buffers
```

Comment:
Looks like it is not true anymore. We could remove the comment.

Reply:
thanks, done